src/LegacyRepContract.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/legacy_reputation/StandardToken.sol';
+import 'ROOT/legacy_reputation/Ownable.sol';
+import 'ROOT/Controller.sol';
+
+
+contract LegacyRepContract is StandardToken, Controlled, Ownable {
+    using SafeMath for uint256;
+
+    event FundedAccount(address indexed _branch, address indexed _sender, uint256 _repBalance, uint256 _timestamp);
+
+    uint256 private constant FAUCET_AMOUNT = 47 ether;
+
+    string public constant name = "Reputation";
+    string public constant symbol = "REP";
+    uint256 public constant decimals = 18;
+
+    uint256 private creation;
+    bool private seeded;
+
+    modifier doneCreating() {
+        require(!isCreating());
+        _;
+    }
+
+    modifier stillCreating() {
+        require(isCreating());
+        _;
+    }
+
+    function LegacyRepContract() {
+        owner = msg.sender;
+        creation = block.timestamp;
+    }
+
+    function isCreating() private returns (bool) {
+        return !seeded || block.timestamp < (creation + 15000);
+    }
+
+    function faucet() public doneCreating returns (bool) {
+        require(balanceOf(msg.sender) == 0);
+        balances[this] = balances[this].sub(FAUCET_AMOUNT);
+        balances[msg.sender] = FAUCET_AMOUNT;
+        FundedAccount(this, msg.sender, FAUCET_AMOUNT, block.timestamp);
+        return true;
+    }
+
+    function transfer(address _to, uint256 _value) public doneCreating returns (bool) {
+        balances[msg.sender] = balances[msg.sender].sub(_value);
+        balances[_to] = balances[_to].add(_value);
+        Transfer(msg.sender, _to, _value);
+        return true;
+    }
+
+    function setSaleDistribution(address[] _addresses, uint256[] _balances) public onlyOwner stillCreating returns (bool) {
```

Comment:
This method can go away, we won't be seeding it with a sale, it'll start with `0` REP and anytime someone calls the faucet function REP will be minted.  `VariableSupplyToken` has a mint function you can call to make the process easy.

Reply:
Right now a good bit of code works by migrating REP from whatever the controller is pointing to at 'LegacyRepContract' onto the ReputationToken. Do we want to refactor this in some fashion?

Reply:
No, that is desirable.  However, that shouldn't have any impact on this contract?  The `LegacyRepContract` returned from the controller can be _any_ ERC20 token.  Just needs an approve and a `transferFrom` method.  We want this contract to be a dead simple ERC20 token that allows anyone to mint REP for themselves for testing.

Reply:
Got it, will do